README.md

```diff
@@ -1,24 +1,96 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
 
-* Ruby version
+## To contribute
+Fork the this repo then clone from forked repo from your github account 
 
-* System dependencies
+```git clone https://github.com/your_username/urbanite.git```  
 
-* Configuration
+or   
+```git clone git@github.com:your_username/urbanite.git```
 
-* Database creation
+### Setting up environment
 
-* Database initialization
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+Our application is dependent on the following:
 
-* Deployment instructions
+- ruby version 2.6.3
 
-* ...
+- rails v 6.0.2
+
+- nodejs v 12 
+
+The installation varies depending on the enviroment/ OS this guide dwells majorly on debian based linux
```

Comment:
The installation varies depending on the environment/OS in use.This guide is dependent on Debian based Linux.